### PR TITLE
Added git and linux commands

### PR DESCRIPTION
**Pull request by rajji007**
Added few git and linux commands

### CODE REVIEW

File: linux_commands.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cd 
+mkdir
+ls
+more/cat - open file in readonly mode
+pwd- present working directory
```
Comment (outrageousaman) [inline linux_commands.txt]: pwd - *

Comment (rajji007) [reply linux_commands.txt]: Done

https://github.com/rajji007/Learning_git/pull/1/commits/f08e64cef5c6fe7d66a285b4c984dace99e7cdc6

Comment (outrageousaman) [reply linux_commands.txt]: changes LGTM.

**Review by outrageousaman — APPROVED** [review]
LGTM

**Pull request merged**